liferay-echo/liferay-portal
# LPS-168270 Usage count for collection for asset libraries

## Motivation

[Link to ticket](https://issues.liferay.com/browse/LPS-168270)

This PR fixes an impedibug found when checking the story to show the usage count of collection in an asset library


## Change summary:

* Add a new finder so that we can retrieve the usage count of collection for a company
* Show the usage count for company in asset library
* Don't show the action for usage count when in asset libraries


## Test Scenario

* Add an asset library
* Create a collection in it
* Connect it to a site
* Use the collection in a site, can be through a collection display
* Go back to the asset library
* Usage count should be reflected

![usages](https://user-images.githubusercontent.com/4267448/201309191-62e9c7f4-7fd3-44b9-ac4e-329bc25e7f88.gif)
